examples/shopping/cart.flow.yaml

```diff
@@ -0,0 +1,279 @@
+import:
+  - products.flow.yaml
+
+collections:
+  - name: examples/shopping/updateCarts
```

Comment:
What throws me a bit is that these collection names are verbs, and not nouns. cartUpdates & cartCheckouts ?